frame/staking/src/pallet/mod.rs

```diff
@@ -1683,13 +1700,16 @@ pub mod pallet {
 			//   threshold bond required.
 			//
 			// Otherwise, if caller is the same as the controller, this is just like `chill`.
+			let used_weight = if let Some(controller) = controller {
+				let ledger = Self::ledger(&controller).ok_or(Error::<T>::NotController)?;
+				let stash = ledger.stash;
+
+				// caller controlls the controller.
```

Comment:
Seems comment is not correct. I believe this condition is to check if Nominator Preference is undecodeable?